app/modules/recipes/recipes.tpl.html

```diff
@@ -1,4 +1,4 @@
-<button ng-click="recipesCtrl.addNewRecipe()">Add my own recipe</button>
+<button class="hidden" ng-click="recipesCtrl.addNewRecipe()">Add my own recipe</button>
```

Comment:
Bootstrap has classes for this, but better way to hide this button - ng-if="false" - to remove it from DOM